framework/jaif-inference-scripts/insert-annotations.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# This script receives as arguments a path to a directory containing .jaif files
+# and a path to a directory containing .java files. It runs the
+# insert-annotations-to-source tool for each file in the directories passed as
+# arguments. It assumes that both sets are in a similar file
+# hierarchy. For example, the $file jaif_path/package/foo.jaif will have its
+# annotations written into $java_source_path/package/foo.java.
+
+function insert_annos_from_jaif() {
+    source_files_dir=$1
+    jaif_files_dir=$2
+    root_dir=`pwd`
+    cd $source_files_dir
+    for f in $(find . -name "*.java"); do
+        f=${f:2}
```

Comment:
I would find comments useful:  stripping leading "./", converting ".java" to ".jaif".